Source/WebCore/platform/ProcessQualified.h

```diff
@@ -56,6 +56,12 @@ class ProcessQualified {
     {
     }
 
+    ProcessQualified(const T& object)
```

Comment:
I'd like to hear from Alex C. about whether this is OK; it might make it easy to make programming mistakes with security consequences.

Reply:
I think it's OK because it matches what we already do for ObjectIdentifier. **HOWEVER, PLEASE MARK AS EXPLICIT!** It is terrible otherwise.

Reply:
I think this actually isn't necessary anymore.